qcfractal/interface/models/rest_models.py

```diff
@@ -46,49 +40,52 @@ def _register_model(is_custom_queries: bool, name: str, rest: str, body: ProtoMo
         The REST query response model.
 
     """
-
-    if is_custom_queries:
-        models_dict = __custom_rest_models
-    else:
-        models_dict = __rest_models
-
-    name = name.lower()
     rest = rest.upper()
 
-    if (name in models_dict) and (rest in models_dict[name]):
+    if (name in __rest_models) and (rest in __rest_models[name]):
         raise KeyError(f"Model name {name} already registered.")
 
-    if name not in models_dict:
-        models_dict[name] = {}
+    if name not in __rest_models:
+        __rest_models[name] = {}
 
-    models_dict[name][rest] = (body, response)
+    __rest_models[name][rest] = (body, response)
 
 
-def rest_model(resource: str, rest: str, subresource: str = None) -> Tuple[ProtoModel, ProtoModel]:
-    """Aquires a REST Model
+def rest_model(resource: str, rest: str) -> Tuple[ProtoModel, ProtoModel]:
+    """
+    Acquires a REST Model.
 
     Parameters
     ----------
     resource : str
         The REST endpoint resource name.
     rest : str
         The REST endpoint type: GET, POST, PUT, DELETE
-    subresource: str
-        A subresource under the main resource
+
     Returns
     -------
     Tuple[ProtoModel, ProtoModel]
         The (body, response) models of the REST request.
 
     """
-    try:
-        if resource.lower() in __custom_rest_models:
-            return __custom_rest_models[resource.lower()][subresource.upper()]
-        else:
-            return __rest_models[resource.lower()][rest.upper()]
-    except KeyError:
-        sub = subresource.lower() if subresource else ''
-        raise KeyError(f"REST Model {rest.upper()} {resource.lower()}:{sub} could not be found.")
+    rest = rest.upper()
+    matches = []
+    for model_re in __rest_models.keys():
```

Comment:
Considering we can get up to 100k REST requests a day right now, does this slow us down appreciably? Might be best to add a `lru_cache`.